scripts/qa_crowbarsetup.sh

```diff
@@ -2365,6 +2365,10 @@ function oncontroller_testsetup()
         fi
     fi
 
+    #test for Glance scrubber service, added after bnc#930739
+    su - glance -s /bin/sh -c "/usr/bin/glance-scrubber --config-dir /etc/glance" || \
+    complain 113 "Glance scrubber doesn't work properly"
```

Comment:
@bmwiedemann glance-scrubber not always return 0, this test it to prevent 

```

2015-05-13 14:50:03.848 22898 CRITICAL glance [-] BadStoreConfiguration: None
2015-05-13 14:50:03.848 22898 TRACE glance Traceback (most recent call last):
2015-05-13 14:50:03.848 22898 TRACE glance   File "/usr/bin/glance-scrubber", line 10, in <module>
2015-05-13 14:50:03.848 22898 TRACE glance     sys.exit(main())
2015-05-13 14:50:03.848 22898 TRACE glance   File "/usr/lib64/python2.6/site-packages/glance/cmd/scrubber.py", line 53, in main
2015-05-13 14:50:03.848 22898 TRACE glance     glance_store.create_stores(config.CONF)
2015-05-13 14:50:03.848 22898 TRACE glance   File "/usr/lib64/python2.6/site-packages/glance_store/backend.py", line 190, in create_stores
2015-05-13 14:50:03.848 22898 TRACE glance     store_instance.configure()
2015-05-13 14:50:03.848 22898 TRACE glance   File "/usr/lib64/python2.6/site-packages/glance_store/_drivers/vmware_datastore.py", line 259, in configure
2015-05-13 14:50:03.848 22898 TRACE glance     self.server_host = self._option_get('vmware_server_host')
2015-05-13 14:50:03.848 22898 TRACE glance   File "/usr/lib64/python2.6/site-packages/glance_store/_drivers/vmware_datastore.py", line 299, in _option_get
2015-05-13 14:50:03.848 22898 TRACE glance     store_name='vmware_datastore', reason=reason)
2015-05-13 14:50:03.848 22898 TRACE glance BadStoreConfiguration: None
2015-05-13 14:50:03.848 22898 TRACE glance
```
`exit code` from this is `1`